server/pages/Home.js

```diff
@@ -38,6 +38,12 @@ function Home(props: Props) {
               try signing in with your company Google account.
             </Notice>
           )}
+          {props.notice === 'hd-not-allowed' && (
+            <Notice>
+              Sorry, your Google apps domain is not allowed. Please try again
+              with an allowed company domain.
```

Comment:
What do you think about printing the allowed domains list in the error response here?

Reply:
Yeah, that might be revealing too much info to an outsider. A real member can contact their admin in any case.